extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -581,68 +598,53 @@ private function doUpdateTags( &$request, $tag_id = false ) {
 	 *  $this->mMassTag
 	 *  $this->mMassTagWikis
 	 *
-	 * @access private
-	 * @author uberfuzzy@wikia-inc.com
 	 * @note yes, i know this is HORRIBLY ineffecient, but I was going for RAD, not clean.
 	 */
 	private function doMassTag() {
+		$this->mMassTagWikis = explode( "\n", $this->mMassTagWikis );
+		$added = 0;
 
-		wfProfileIn( __METHOD__ );
-
-		$this->mMassTagWikis = explode("\n", $this->mMassTagWikis);
-		global $wgOut;
-
-		$msg  = false;
-		$added=0;
-
-		foreach($this->mMassTagWikis as $dart_string)
-		{
-			if( empty($dart_string) ) { continue; }
-			$dart_parts = explode("/", $dart_string);
-			if( count($dart_parts) < 2 ) {
+		foreach ( $this->mMassTagWikis as $dart_string ) {
+			if ( empty( $dart_string ) ) { continue; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)